### ISSUE TITLE
Post Deploy Error - "msg": "********sdk is required for this module"

### ISSUE THREAD

**Issue by tislam93** [ChameleonCloud/chi-in-a-box#104]
The post-deploy step shows error when it reaches the task "precis : Create cep service"
```
PLAY [precis] **********************************************************************************************************************************************************

TASK [precis : Create cep service] *************************************************************************************************************************************
fatal: [ara-sb-03]: FAILED! => {"changed": false, "msg": "********sdk is required for this module"}
```
I am not sure why the sdk name is hidden and which service (or tool) needs to be installed manually (or mentioned in any file) for this step to be performed successfully? Some workarounds (where the error mentions "openstacksdk") says ```pip install future``` should resolve the issue but it didn't work in this case.

**Comment by tislam93** [comment]
I tried various workarounds with this issue but could not succeed. Right now, I tried to bypass the precis playbook by mentioning ``` enable_precis: no ``` in globals.yml but still got the same error. 
I was wondering how can I bypass this step (if at least I can not solve it for the time being)? And if some changes are made to the globals file, which services (specific tags) needs to be reconfigured with the ``` ./cc_ansible reconfigure --tags ... ``` command?
